README.md

```diff
@@ -8,7 +8,7 @@ The tag reader is a simple to build/use NFC tag reader, specially created for [H
 
 > I am selling a pre-built version, a DIY version (assembly required) or just the case (use with own components). Check it out on [my website](https://adonno-crafts.myshopify.com/).
 
-![Photos of the final product](https://raw.githubusercontent.com/adonno/tagreader/master/docs/cases.jpg)
```

Comment:
Would be helpful if we get unrelated changes in a separate PR in case this PR takes a while to complete.

Reply:
#46